### PR TITLE
Add format for the meetup

### LINKED ISSUE #16: Create a format for the Opf Data Science Meetup
To ensure consistency across all meetups, plan and design a format for each Meetup.

For example - 

1. Opening and Introduction of the Speaker 
2. Presentation 
3. Q&A
4. Closing Note.

### PR DESCRIPTION
**Pull request by aakankshaduggal**
Resolves #16

### CODE REVIEW

File: docs/format.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+After the presentation, take questions from the attendees. Make sure all the questions that were asked during/after the session in chat are also answered.
+
+### Closing Note
+
+Thank the speaker and all the attendees for taking out some time to attend the meetup and talk a little about the upcoming session.
+
+## Before the session 
+
+* Send out an email invite to certain target groups, briefing them about the Operate First Data Science Community Meetup.
```
Comment (oindrillac) [inline docs/format.md:27]:
```suggestion
* Send out an email invite to certain target groups, briefing them about the upcoming Operate First Data Science Community Meetup.
```

File: docs/format.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+* Send out a tweet from the operate-first Twitter account by adding a PR [here](https://github.com/operate-first/operate-first-twitter).
+
+## After the session
+
+* Edit the video recording and post it to Operate First [YouTube Channel.](https://youtube.com/playlist?list=PL8VBRDTElCWqC5WcZUUNJxGn9WslK6llb)
+* Export the list of attendees to google drive.
```
Comment (oindrillac) [inline docs/format.md:33]: is there a particular drive this is being added to?

File: docs/format.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+* Send out an email updating the attendees about the presentation and the video recording about the Operate First Data Science Community Meetup.
```
Comment (oindrillac) [inline docs/format.md:34]:
```suggestion
* Send out an email updating the attendees about the presentation and the video recording of the previous Operate First Data Science Community Meetup.
```

File: docs/format.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+* Send out a tweet from the operate-first Twitter account with the video link by adding a PR [here](https://github.com/operate-first/operate-first-twitter).
```
Comment (oindrillac) [inline docs/format.md:35]:
```suggestion
* Send out a tweet from the operate-first Twitter account with the previous meetup's video link by adding a PR [here](https://github.com/operate-first/operate-first-twitter).
```

**Review by oindrillac — COMMENTED** [review]
Looks good! Some small nits

File: docs/format.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# Format for Operate First Data Science Meetup
+
+## During the session
+
+### Opening and Welcome Note 
+
+Start the session with a warm welcome to all the attendees. Make sure to hit the record button to have a recording for the session.
```
[the remark this answers is not in the document]
Comment (aakankshaduggal) [reply docs/format.md:7]: Yeah, that's a good idea actually. We could at least let them while we are working on that policy.

File: docs/format.md (hunk 2 above)
Comment (aakankshaduggal) [reply docs/format.md:33]: Currently, it gets to Michael's drive since he is the organizer of this event.

**Review by chauhankaranraj — CHANGES_REQUESTED** [review]

File: docs/format.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+### Closing Note
+
+Thank the speaker and all the attendees for taking out some time to attend the meetup and talk a little about the upcoming session.
+
+## Before the session 
+
+* Send out an email invite to certain target groups, briefing them about the upcoming Operate First Data Science Community Meetup.
+* Send out a tweet from the operate-first Twitter account by adding a PR [here](https://github.com/operate-first/operate-first-twitter).
+
```
[the remark this answers is not in the document]
Comment (aakankshaduggal) [reply docs/format.md:29]: Sure, the idea behind the current flow is to draw attention to the format of the meetup presentation. 
Before and After are mainly chores that need to be done in order to promote and invite the attendees. 
Wdyt?

[the remark this answers is not in the document]
Comment (chauhankaranraj) [reply docs/format.md:29]: Ah okay if that's the case then this works just fine :)

**Review by chauhankaranraj — COMMENTED** [review]
/lgtm

**Pull request merged**